index.html

```diff
@@ -2,159 +2,39 @@
-        <td>Defines a table caption</td>
-      </tr>
-      <tr>
-        <td>&#60;thead&#62;</td>
-        <td>Groups the header content in a table</td>
-      </tr>
-      <tr>
-        <td>&#60;tbody&#62;</td>
-        <td>Groups the body content in a table</td>
-      </tr>
-      <tr>
-        <td>&#60;tfoot&#62;</td>
-        <td>Groops the footer content in a table</td>
-      </tr>
-    </table>
-    <br />
-    <form action="/" method="get" autocomplete="on">
-      <fieldset>
-        <legend>My first form</legend>
-        <label for="name">Name</label>
-        <br />
-        <input id="name" type="text" name="name" placeholder="Name" required />
-        <br />
-        <label for="email">E-mail</label>
-        <br />
-        <input id="email" type="email" name="email" required />
-        <br />
-        <label for="password">Password</label>
-        <br />
-        <input id="password" type="password" name="password" required />
-        <br />
-        <label for="level">Your programming level</label>
-        <br />
-        <input id="level" type="range" name="level" min="1" max="10" />
-        <p>What does HTML stand for?</p>
-        <input id="html-1" type="checkbox" name="html" value="1" />
-        <label for="html-1">Hyperlinks and Text Markup Language</label><br />
-        <input id="html-2" type="checkbox" name="html" value="2" />
-        <label for="html-2">Home Tool Markup Language</label><br />
-        <input id="html-3" type="checkbox" name="html" value="3" />
-        <label for="html-3">Hyper Text Markup Language</label>
-        <p>What does HTTP stand for?</p>
-        <input id="htmp-1" type="radio" name="http" value="1" />
-        <label for="htmp-1">Hyper Text Translate Protocol</label><br />
-        <input id="htmp-2" type="radio" name="http" value="2" />
-        <label for="htmp-2">Hyper Text Transmission Protocol</label><br />
-        <input id="htmp-3" type="radio" name="http" value="3" />
-        <label for="htmp-3">Hyper Text Transfer Protocol</label><br />
-        <label for="comment">Comment</label><br />
-        <textarea
-          id="comment"
-          name="comment"
-          placeholder="Comment"
-          maxlength="500"
-        ></textarea>
-        <br />
-        <label for="select_html">What does HTML stand for?</label><br />
-        <select id="select_html" name="select_html">
-          <option disabled selected>Your answer</option>
-          <option value="1">1</option>
-          <option value="2">2</option>
-          <option value="3">3</option>
-        </select>
-        <p>
-          <button>Send</button>
-        </p>
-      </fieldset>
-    </form>
+    <div class="xl-container-advantages">
+      <h3>What do we offer</h3>
+      <hr />
+      <div class="xs-container-advantages flex-center-evenly">
+        <div class="advantages-content">
+          <div class="advantages-img">
+            <img src="./images/support.png" alt="Support 24/7" />
```

Comment:
this image don`t have any styles, it`s not a problem for now... but would be better to add styles for picture